README.md

```diff
@@ -42,7 +42,11 @@ represent the tiles.
 
 ## A1.1
 
-«replace this with your answer»
+S -> ASA
+S -> A
+A -> "r"
+A -> "g"
+A -> "b"
```

Comment:
You're on the right track, but the two A's in the first line don't have to have the same value. (if they did, a grammar rule such as `number = digit+` would only accept numbers where all the digits were the same.

You need something like:

~~~
       S -> r | g | b
       S -> r  S  r
       S -> g  S  g
       S -> b  S  b
~~~

# -3